api/src/repositories/spatial-repository.ts

```diff
@@ -297,6 +297,21 @@ export class SpatialRepository extends BaseRepository {
     return response.rows;
   }
 
+  async deleteSpatialComponentsBySubmissionId(submission_id: number): Promise<{ submission_id: number }[]> {
+    const sqlStatement = SQL`
```

Comment:
@NickPhura Ended up modifying my approach from `getKnexQueryBuilder` to simply using raw SQL, since the query builder response was returning `undefined`.